weekly-standup-template.md

```diff
@@ -264,6 +264,29 @@ For example, this may be an emergent task, providing support or expertise to oth
 
 #### Other
 
+##### [Promotion, opportunities and new work venues](https://github.com/alan-turing-institute/data-safe-haven-team/issues/68)
+
+PIs update here!
+
+###### People and time
+
+<!--
+- Harry Lime (x hours)
+- Sam Space (y hours)
```

Comment:
Just realised I put Sam Space everywhere instead of Sam Spade 🤦.

Reply:
I thought it was cool